### PR TITLE
'express'

### PR DESCRIPTION
**Pull request by Siarhei-Balonikau**

### CODE REVIEW

File: http-servers/echo-server.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var http = require('http');
+var fs = require("fs");
+var filename = "./index.html";
+
+
+http.createServer()
+.on('request', function (req, res) {
```
Comment (slowpokex) [inline http-servers/echo-server.js]: (req, res) => req.pipe(res)
Please prefer arrow functions

File: src/controllers/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+exports.getAllProducts = function(req, res) {
```
Comment (slowpokex) [inline src/controllers/index.js]: please convert all to arrow functions

File: src/middlewares/parse-query.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const parseQuery = function (req, res, next) {
```
Comment (slowpokex) [inline src/middlewares/parse-query.js]: parse(req.url, true)

File: src/routes/index.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import controllers from './../controllers/index.js';
+
+const router = express.Router();
+
+router.get('/products', controllers.getAllProducts);
```
Comment (slowpokex) [inline src/routes/index.js]: Please chain it

**Review by slowpokex — CHANGES_REQUESTED** [review]
Please correct your hometask

File: http-servers/echo-server.js (hunk 1 above)
Comment (Siarhei-Balonikau) [reply http-servers/echo-server.js]: Done

File: src/controllers/index.js (hunk 2 above)
Comment (Siarhei-Balonikau) [reply src/controllers/index.js]: Done

File: src/routes/index.js (hunk 4 above)
Comment (Siarhei-Balonikau) [reply src/routes/index.js]: Done

**Pull request merged**